.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,77 @@
+name: Bug Report
+description: Report bug or performance issue
+title: "BUG: "
```

Comment:
In my opinions if we already have labels it is a bit redundant.

Then again it squares with the concommits.